lab_3_ann_retriever/main.py

```diff
@@ -157,6 +193,13 @@ def build(self) -> bool:
         Returns:
             bool: True if built successfully, False in other case
         """
+        if self._corpus is None or len(self._corpus) == 0:
+            return False
+
+        self._vocabulary = sorted(build_vocabulary(self._corpus))
+        self._idf_values = calculate_idf(self._vocabulary, self._corpus)
+        self._token2ind = {token: self._vocabulary.index(token) for token in self._vocabulary}
+        return True if self._vocabulary and self._idf_values and self._token2ind else False
```

Comment:
the same as `bool(self._vocabulary and self._idf_values and self._token2ind)`

Reply:
done